manager/controllers/app/fybrikapplication_conditions.go

```diff
@@ -43,21 +46,27 @@ func resetAssetState(application *api.FybrikApplication, assetID string) {
 	application.Status.AssetStates[assetID] = api.AssetState{Conditions: conditions}
 }
 
-func setErrorCondition(application *api.FybrikApplication, assetID string, msg string) {
+func setErrorCondition(log zerolog.Logger, application *api.FybrikApplication, assetID string, err error, msg string) {
```

Comment:
In case of an error, msg already contains it. Why do we need both err and msg?

Reply:
in checkReadiness(), there are cases where there is just a message and no err.
In those cases the err parameter is nil.

Reply:
Right. What I mean is that err is not required. Or you could go over the messages and remove err.Error() from them. Otherwise we'd see the err.Error() printed twice in the log.